### PR TITLE
[YUNIKORN-2836] Simplify YuniKorn version selection in Getting Started page

### PR DESCRIPTION
**Pull request by blueBlue0102**
### What is this PR for?
This PR simplifies YuniKorn version installation by replacing custom YAML with a single helm install command using the `--version` flag. It streamlines the process for easier user adoption.

### What type of PR is it?
* [x] - Documentation

### What is the Jira issue?
[YUNIKORN-2836](https://issues.apache.org/jira/browse/YUNIKORN-2836)

### CODE REVIEW

**Comment by chenyulin0719** [comment]
@blueBlue0102 Could you add a note to inform users that there is no 'latest' version available on DockerHub? Users can check [this link](https://hub.docker.com/r/apache/yunikorn/tags) to see the available tag versions.
- https://yunikorn.apache.org/docs/next/user_guide/service_config#helm-configuration

File: docs/get_started/get_started.md [hunk 1]
```diff
@@ -50,23 +50,9 @@ admission controller delegating all scheduling to YuniKorn. Because this mode is
...
-  image:
-    tag: admission-{version}  # default depends on YuniKorn version
-
-web:
-  image:
-    tag: web-{version}        # default depends on YuniKorn version
+helm install yunikorn yunikorn/yunikorn --namespace yunikorn --version {version}
 ```
 
 You can check available `{version}` on [DockerHub](https://hub.docker.com/r/apache/yunikorn/tags).
```
Comment (chenyulin0719) [inline docs/get_started/get_started.md:58]: The available chart version is on this page:
- https://artifacthub.io/packages/helm/yunikorn/yunikorn

File: docs/get_started/get_started.md [hunk 2]
```diff
@@ -50,26 +50,12 @@ admission controller delegating all scheduling to YuniKorn. Because this mode is
...
-  image:
-    tag: admission-{version}  # default depends on YuniKorn version
-
-web:
-  image:
-    tag: web-{version}        # default depends on YuniKorn version
-```
-
-You can check available `{version}` on [DockerHub](https://hub.docker.com/r/apache/yunikorn/tags).
+You can check available `{version}` on [here](https://artifacthub.io/packages/helm/yunikorn/yunikorn).
```
Comment (chenyulin0719) [inline docs/get_started/get_started.md:58]: nit: I'm not in favor of highlighting `{version}` here.  Could we remove the highlight? 

The description for Docker version is relatively simple and clear.

image

**Review by chenyulin0719 — APPROVED** [review]
LGTM

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**